### DISCUSSION TITLE
Migrate to supabase auth?

### DISCUSSION THREAD

**Discussion by proko** [supabase/supabase#532] [Questions]
Hello! I'm very interested in migrating a project I'm working on to supabase but when it comes to importing users, it looks like there is not yet an oficial guide available. Is there really a "supabase" way or should I be looking for a "postgresql" way of doing it instead? Currently  the users are stored in faunaDB in case it makes any difference. Any resources on the topic would be very much appreciated since I'm not too familiar with DB administration.

**Comment by EryouHao** [comment]
I am also facing this question and hope someone can help answer it. Hi @kiwicopple , Do you have any ideas about this issue?

**Comment by silentworks** [comment]
This could be done similar to what I explained in similar question asked by another user https://github.com/supabase/supabase/discussions/1264#discussioncomment-656608.
